tf2_ros/include/tf2_ros/transform_listener.h

```diff
@@ -133,13 +133,12 @@ class TransformListener
       // Duplicate to modify option of subscription
       rclcpp::SubscriptionOptionsWithAllocator<AllocatorT> tf_options = options;
       rclcpp::SubscriptionOptionsWithAllocator<AllocatorT> tf_static_options = static_options;
-      tf_options.callback_group = callback_group_;
-      tf_static_options.callback_group = callback_group_;
 
       message_subscription_tf_ = rclcpp::create_subscription<tf2_msgs::msg::TFMessage>(
-        node, "/tf", qos, std::move(cb), tf_options);
+        node, "/tf", qos, std::move(cb), tf_options, callback_group_);
```

Comment:
```suggestion
        node, "/tf", qos, std::move(cb), options, callback_group_);
```

The `tf_options` is no longer used.